YAFC/Windows/WelcomeScreen.cs

```diff
@@ -61,17 +62,35 @@ private enum EditType {
             Workspace, Factorio, Mods
         }
 
-        public WelcomeScreen() : base(ImGuiUtils.DefaultScreenPadding) {
+        public WelcomeScreen(string[] args = null) : base(ImGuiUtils.DefaultScreenPadding) {
+            if (tips == null) {
+                tips = File.ReadAllLines("Data/Tips.txt");
+            }
+
+            IconCollection.ClearCustomIcons();
             RenderingUtils.SetColorScheme(Preferences.Instance.darkMode);
-            var lastProject = Preferences.Instance.recentProjects.FirstOrDefault();
+
+            if (args != null && args.Length >= 3) {
```

Comment:
if only 1 or 2 arguments are given it would be nice to return an error that something is missing. Now it is silently ignored (but the user was trying to do something)

Reply:
I see lots of logs in the console (Linux), let me find how those are logged. I suppose this should also work on Windows (if not we have a new bugreport :smile: )

Reply:
Found it:
https://github.com/have-fun-was-taken/yafc-ce/blob/3f5200a2fa5db6ff913a0cc0c2b56dd39d81dc05/YAFCparser/LuaContext.cs#L315

and a few lines lower an error (I never saw this though, but it looks like it writes to console)
https://github.com/have-fun-was-taken/yafc-ce/blob/3f5200a2fa5db6ff913a0cc0c2b56dd39d81dc05/YAFCparser/LuaContext.cs#L328